### PR TITLE
Doc/add faq section

### PR DESCRIPTION
**Pull request by PipKat**
Added the faq section and did some basic clean up that begins to align the doc with the Google developer style guidelines.

### CODE REVIEW

**Comment by PipKat** [comment]
@dnwillia I've incorporated the info from the FAQ PPT that Nima Bohlooli Arkhazloo shared with me as a new page in the "Getting started" section of the PyFluent documentation. I did have to convert it from the bulleted list format in the original PPT source to the complete sentence style required by the Google developer style guide and Vale style checks. I also have more installation information from a PPT shared with me by Pablo Aguado but will consider adding content from it to the exisiting installing in the PyFluent documentation in a separate PR next week.

File: examples/00-fluent/fluent-20221130-110834-232.trn [hunk 1]
```diff
@@ -0,0 +1,21 @@
+Auto-Transcript Start Time:  11:08:34, 30 Nov 2022 Eastern Standard Time
```
[the remark this answers is not in the document]
Comment (PipKat) [reply examples/00-fluent/fluent-20221130-110834-232.trn]: I removed the TRN files. However, I didn't know how to add add a .trn file entry to gitignore! Sorry!

[the remark this answers is not in the document]
Comment (dnwillia-work) [reply examples/00-fluent/fluent-20221130-110834-232.trn]: At the bottom of the .gitignore file:

https://github.com/pyansys/pyfluent/blob/c9a9069fa9d84136232fb848989712c820f34b49/.gitignore#L143

just add two lines like this:
```
# Fluent generated files
*.trn
```

**Review by dnwillia-work — APPROVED** [review]

File: doc/source/getting_started/faqs.rst [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  :width: 800
+  :alt: PyAnsys overview
+
+What is PyFluent?
+-----------------
+PyFluent provides Pythonic access to Ansys Fluent. Its features enable the seamless use of
+Fluent within the Python ecosystem and broad access to native Fluent features for performing
+actions such as these:
+
+- Launch Fluent using a local Ansys installation.
```
Comment (seanpearsonuk) [inline doc/source/getting_started/faqs.rst:23]: @PipKat @dnwillia-work I realise that this is also the current wording. The fact that one can also connect to a remote instance seems worth a mention here.

File: doc/source/getting_started/faqs.rst [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+- Use Fluent's TUI (text user interface) commands for both meshing and solver features.
+- Use Fluent's built-in postprocessing capabilities.
+
+PyFluent is not bundled with the Fluent installation. You must download and install PyFluent
```
Comment (seanpearsonuk) [inline doc/source/getting_started/faqs.rst:27]: @PipKat @dnwillia-work this statement is also out of date now.

File: doc/source/getting_started/faqs.rst [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+
+UDFs continue to be written in C and remain important elements of
+Fluent simulations.
+
+While you cannot write UDFs in Python, you can execute PyFluent commands to compile and load UDFs,
+similar to how you use TUI commands.
+
+Who should use PyFluent?
+------------------------
+Many different types of people benefit from using PyFluent to achieve various objectives. Users
```
Comment (seanpearsonuk) [inline doc/source/getting_started/faqs.rst:47]: @PipKat @dnwillia-work Original wording again. "Different types of people" is strange phrasing.

File: doc/source/getting_started/faqs.rst [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+include engineers, product designers, partners, consultants, researchers, professors, students,
```
Comment (seanpearsonuk) [inline doc/source/getting_started/faqs.rst:48]: If a list starts with engineers and ends with non-engineers then everything in between is redundant.

File: doc/source/getting_started/faqs.rst [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+  :width: 800
+  :alt: PyFluent users and objectives
+
+
+Here are some typical objectives that you can use PyFluent to achieve:
+
+- Enhance productivity with customized scripts.
+- Automate workflows with customized scripts.
+- Extend CFD simulations to a wider audience.
+- Create comprehensive workflows inspired by Python's increasingly broad offerings
```
Comment (seanpearsonuk) [inline doc/source/getting_started/faqs.rst:62]: Only the final bullet omits the full stop. Deliberate?

File: doc/source/getting_started/faqs.rst [hunk 7]
```diff
@@ -0,0 +1,235 @@
...
+  in these areas of scientific computing:
+
+  - Computer vision
+  - ML (machine learning)
+  - AI (artificial intelligence)
```
Comment (seanpearsonuk) [inline doc/source/getting_started/faqs.rst:67]: Are we able to demonstrate these as three distinct areas where PyFluent could be used?

**Comment by PipKat** [comment]
@seanpearsonuk All excellent points! I've added Ryan O'Conner as a reviewer as he and Nima supplied the basic content. Nima doesn't seem to have access, but Ryan should be able to review and comment.

File: doc/source/user_guide/general_settings.rst [hunk 8]
```diff
@@ -1,9 +1,9 @@
-Applying general settings
+Apply general settings
 =========================
```
Comment (dnwillia-work) [inline doc/source/user_guide/general_settings.rst:2]: This heading underlining is a bit long now?

File: doc/source/getting_started/index.rst [hunk 9]
```diff
@@ -3,94 +3,20 @@
 ===============
 Getting started
 ===============
+
 PyFluent provides Pythonic access to Ansys Fluent. 
 
-To run PyFluent, you must have a licensed copy of Ansys Fluent
-installed locally. PyFluent supports Fluent 2022 R2 and later.
+To run PyFluent, you must have a licensed copy of Fluent 2022 or later
```
Comment (dnwillia-work) [inline doc/source/getting_started/index.rst:9]:
```suggestion
To run PyFluent, you must have a licensed copy of Fluent 2022 R2 or later
```

File: doc/source/getting_started/installation.rst [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+  solver.health_check_service.is_serving
+
+To locate the Fluent installation, PyFluent automatically uses the ``AWP_ROOT<ver>``
+environment variable, where ``<ver>`` is the three-digit format for the release.
+For example, ``AWP_ROOT222`` is the environment variable for the 2022 R2 release. 
+
+On a Windows system, this environment variable is configured when a release is installed.
+
+On a Linux system, you must configure this environment variable to point to the absolute
+path of the installed release. For example, for the 2022 R2 release, you would likely set
```
Comment (dnwillia-work) [inline doc/source/getting_started/installation.rst:59]: This paragraph is about Linux right so the C: path at the end makes no sense.

```suggestion
path of the installed release. For example, for the 2022 R2 release, you might set
```

File: doc/source/getting_started/installation.rst [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+the ``AWP_ROOT222`` environment variable to point to ``C:\Program Files\ANSYS Inc\v222``.
+
+To use a non-default installation location, either set the ``AWP_ROOT<ver>`` environment variable
```
Comment (dnwillia-work) [inline doc/source/getting_started/installation.rst:62]: I think we should take this paragraph out.  It was added for the use case where Fluent developers have a local Fluent build area.

File: doc/source/getting_started/faqs.rst (hunk 2 above)
Comment (dnwillia-work) [reply doc/source/getting_started/faqs.rst:23]: Maybe propose a bullet point wording.

File: doc/source/getting_started/faqs.rst (hunk 3 above)
Comment (dnwillia-work) [reply doc/source/getting_started/faqs.rst:27]: Yeah, where is it sitting in the installation?

File: doc/source/getting_started/faqs.rst (hunk 2 above)
Comment (PipKat) [reply doc/source/getting_started/faqs.rst:23]: I added another bullet about connecting to a Fluent instance running on a remote machine.

File: doc/source/getting_started/faqs.rst [hunk 12]
```diff
@@ -0,0 +1,235 @@
...
+separately. For more information, see :ref:`faqs_install`, which appears later on this page.
+
+PyFluent has no GUI (graphical user interface). You interact with PyFluent through the Python
+environment of your choice.
+
+How does PyFluent compare to UDFs?
+----------------------------------
+PyFluent is conceptually aligned with Fluent TUI console commands (and journaling) rather than with
+UDFs (user-defined functions). In other words, PyFluent is used for automation rather than
+modifying the solver behavior.
```
[the remark this answers is not in the document]
Comment (PipKat) [reply doc/source/getting_started/faqs.rst:38]: I will need a SME to reword this as desired and resolve this conversation.

File: doc/source/getting_started/faqs.rst (hunk 7 above)
Comment (PipKat) [reply doc/source/getting_started/faqs.rst:63]: I will need a SME to address this.

File: doc/source/getting_started/faqs.rst [hunk 13]
```diff
@@ -0,0 +1,231 @@
...
+
+UDFs continue to be written in C and remain important elements of
+Fluent simulations.
+
+While you cannot write UDFs in Python, you can execute PyFluent commands to compile and load UDFs,
+similar to how you use TUI commands.
+
+Who should use PyFluent?
+------------------------
+PyFluent users include engineers, product designers, consultants, and academia.
```
Comment (PipKat) [inline doc/source/getting_started/faqs.rst:48]: @seanpearsonuk Should we remove this image or have whoever owns the source to update it? Images do often become maintenance issues long-term....

File: doc/source/getting_started/faqs.rst (hunk 3 above)
Comment (PipKat) [reply doc/source/getting_started/faqs.rst]: @seanpearsonuk @dnwillia-work I noted that it was included--but not the where. I can add this information if someone provides it.

**Comment by PipKat** [comment]
@seanpearsonuk @dnwillia-work Any change of getting the remaining conversations resolved and this PR merged soon?
@rploconnor Do you have time to do a review and approval?

File: doc/source/getting_started/faqs.rst (hunk 12 above)
[the remark this answers is not in the document]
Comment (dnwillia-work) [reply doc/source/getting_started/faqs.rst:38]: Yeah, that makes sense.

**Review by dnwillia-work — APPROVED** [review]

**Review by seanpearsonuk — APPROVED** [review]
